parsers/ELEXON.py

```diff
@@ -218,6 +224,48 @@ def datetime_from_date_sp(date, sp):
     return datetime.replace(tzinfo='Europe/London').datetime
 
 
+def _fetch_wind(target_datetime=None):
+    if target_datetime is None:
+        target_datetime = dt.datetime.now()
+    start = target_datetime - dt.timedelta(hours=12)
+    end = target_datetime + dt.timedelta(hours=12)
+
+    session = requests.session()
+    params = {
+        'FromDateTime': start.strftime('%Y-%m-%d %H:%M:%S'),
+        'ToDateTime': end.strftime('%Y-%m-%d %H:%M:%S'),
+        'ServiceType': 'csv'
+    }
+    response = query_ELEXON('FUELINST', session, params)
+    csv_text = response.content
```

Comment:
I think if this is changed to `csv_text = response.text` it should fix the error.